6-Typescript/server/controllers/todoController.ts

```diff
@@ -1,78 +1,66 @@
-import { ITodo } from './../interfaces/ITodo';
-import {Request, Response} from 'express'
-import { ITodoService } from '../interfaces/ITodoService';
+import { ITodo } from "../interfaces/interfaces";
+import { Request, Response } from "express";
+import { ITodoService } from "../interfaces/interfaces";
 export class TodoController {
-    db: ITodoService
+    private db: ITodoService;
     constructor(db: ITodoService) {
-        this.db = db
-
+        this.db = db;
     }
     getTodoById = async (req: Request, res: Response): Promise<void> => {
         const userId: string = req.body.userId;
-        const todoId: string= req.params.id
-        
-        const todo: ITodo | null = await this.db.modelGetTodoById(userId, todoId);
-        if(todo) {
-            res.status(200).json(todo)
-        } else {                                                
-            res.sendStatus(404)
+        const todoId: string = req.params.id;
+
+        const todo: ITodo | null = await this.db.modelGetTodoById(
+            userId,
+            todoId
+        );
+        if (todo == null) {
+            res.sendStatus(404);
+            return;
         }
-    }
-    postTodo = async (req: Request, res: Response): Promise<void> => {   
+        res.status(200).json(todo);
+    };
+    postTodo = async (req: Request, res: Response): Promise<void> => {
+        const newTodo = req.body;
```

Comment:
do we know the type of `newTodo`?

Reply:
Sure i guess i can make the req.body and write:
req.body as ITodo.
Is that what you meant?